makefile

```diff
@@ -15,7 +15,7 @@ hash:
 	$(GPP) -m32 -O2 -fshort-wchar -shared -o $(OUTPUT) *.o $(LIBRARIES)
 	
 cryptolib:
-	$(MAKE) -C $(CRYPTOPP_SRC_DIR) static
+	CXXFLAGS="-DNDEBUG -m32 -g2 -O3 -fPIC" $(MAKE) -C $(CRYPTOPP_SRC_DIR) static
```

Comment:
It builds crypto++ in 32-bit. That's why the compiler says incompatible with crypto++ after build.